.circleci/config.yml

```diff
@@ -122,9 +129,22 @@ references:
 
             # Build wheels
             for PYBIN in /opt/python/*/bin; do
-                echo "Building for `${PYBIN}/python --version`"
-                "${PYBIN}/pip" install -r ~/ci/freud/requirements.txt --progress-bar=off
-                "${PYBIN}/pip" wheel ~/ci/freud/ -w ~/wheelhouse/ --no-deps
+              # Split the echo command and the version since python2
+              # --version doesn't return the value, just prints it.
+              echo "Building for "
+              ${PYBIN}/python --version
+
+              "${PYBIN}/python" -m pip install cython --no-deps --ignore-installed -q --progress-bar=off
+              rm -rf numpy-1.10.0rc2
+              curl -sSLO https://github.com/numpy/numpy/archive/v1.10.0rc2.tar.gz
```

Comment:
I would prefer to use the last bugfix release of 1.10; 1.10.4 instead of 1.10.0(rc2). https://github.com/numpy/numpy/releases/tag/v1.10.4

You can see the bugfixes between 1.10.0 and 1.10.4 here: https://docs.scipy.org/doc/numpy-1.11.0/release.html

Reply:
Good call, for some reason I decided to initially go with the earliest release but the ABI shouldn't change so might as well use the newest bugfix release.